src/app/score/Score.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { Actions } from './actions/Actions';
+import { Congratulations } from './Congratulations';
+import { Results } from './Results';
+import { Styled } from './styled';
+
+export const Score: React.FC = () => {
```

Comment:
If you don't have props, component could be simplified:
`export const Score = () =>...`